tests/mediaembedui.js

```diff
@@ -53,6 +53,12 @@ describe( 'MediaEmbedUI', () => {
 		expect( dropdown ).to.be.instanceOf( DropdownView );
 	} );
 
+	it( 'should allow creating two instances', () => {
```

Comment:
To make this test a bit more bulletproof, it'd be good to ensure that two instances are actually created. What if someone changes the beforeEach to not create this item (I can imagine this line being moved to particular it()s?

So, I'd simply call this `.create()` twice in this test.

It'd also be good to verify that it's not the same instance returned twice :D